src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/UnusedImportsCheck.java

```diff
@@ -65,6 +65,10 @@
  * comment {@code {@link List}}. The alternative to avoid introducing a compile-time
  * dependency would be to write the Javadoc comment as {@code {&#64;link java.util.List}}.
  * </li>
+ * <li>
+ * A static method is imported when used as method reference. In that case,
```

Comment:
Please move this item above, above "Optionally"

Reply:
done